elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -18,8 +27,8 @@ def render(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     # get the name of the element, in this case, the name of the array
     name = pl.get_string_attrib(element, 'answers-name')
-    label = pl.get_string_attrib(element, 'label', None)
-    allow_partial_credit = pl.get_boolean_attrib(element, 'allow-partial-credit', False)
+    label = pl.get_string_attrib(element, 'label', LABEL_DEFAULT)
+    allow_partial_credit = pl.get_boolean_attrib(element, 'allow-partial-credit', ALLOW_PARTIAL_CREDIT_DEFAULT)
```

Comment:
Should a default be set for `allow-feedback`? It is piggybacking off of `allow-partial-credit` for `pl-matrix-component-input`.

Reply:
@tbretl ^^^

Reply:
Given the structure in the docs, I think this was by design. We should probably let this be until the course-wide defaults are in place.